src/actions/userActions.ts

```diff
@@ -60,4 +71,10 @@ export namespace User {
       ...new Actions.Logout(),
     });
   };
+
+  export const changeOrg = (orgId: number) => (dispatch: Dispatch<Actions.ChangeOrg>) => {
+    dispatch({
+      ...new Actions.ChangeOrg({ orgId })
```

Comment:
Just an eslint thing.

```suggestion
      ...new Actions.ChangeOrg({ orgId }),
```